package.json

```diff
@@ -14,7 +18,8 @@
     "react-router": "^6.2.1",
     "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
-    "web-vitals": "^2.1.3"
+    "styled-components": "^5.3.3",
```

Comment:
Are you using this somewhere, if not remove all unnecessary packages from package.json